modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
 [NOTE]
 Run performance and load testing on your Mule applications to determine the number of resources to allocate.
 
+== Internal Load Balancer Memory Allocation
+
+Internal load balancer memory requirements are impacted by the number of threads, response time latency, and message sizes. The following internal load balancer memory allocations are recommended:
+
+* .5 GB (Default): Fewer than 500 simultaneous active connections.
+* 1.5 GB (Large): One or both of the following scenarios:
```

Comment:
Could you recast this guideline as a sentence, such as this one: "Allocate 1.5 GB of memory if one or both of these conditions are met:
* The load balancer handles 500 or more active connections simultaneously.
* Security policies are enabled for the load balancer."?